cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -272,6 +279,34 @@ def upload_fusion_report_to_scout(
 
         LOG.info(f"Upload {report_type} fusion report finished!")
 
+    def upload_rna_delivery_report_to_scout(self, dry_run: bool, case_id: str) -> None:
```

Comment:
If you have time it would be great to add a test for this function. You could mimic the test `test_upload_rna_report_to_successful_dna_case_in_scout` or `test_upload_research_rna_fusion_report_to_scout` in `tests/meta/upload/scout/test_meta_upload_scoutapi_rna.py`